client/my-sites/checkout/composite-checkout/components/checkout-next-steps.tsx

```diff
@@ -77,35 +76,33 @@ const NextStepIcon = styled( BaseIcon )`
 export default function CheckoutNextSteps( { responseCart, headerText }: Props ) {
 	const translate = useTranslate();
 
-	const steps: NextStepsStep[] = useMemo( () => {
-		if ( hasDIFMProduct( responseCart ) ) {
```

Comment:
This component was meant to be reusable, so this box can also be utilized for other products. For example:
```
if product is BBE:
  steps = [....]
else if product is somethingElse:
  steps = [...]
```

So maybe we can keep this component as it is? If not, then this component should be renamed to `CheckoutBBENextSteps` or similar.

Reply:
Ah right this makes sense 👍 I will change this!